src/cart/crt_rpc.c

```diff
@@ -1112,22 +1116,32 @@ crt_req_ep_lc_lookup(struct crt_rpc_priv *rpc_priv, bool *uri_exists)
 	*uri_exists = false;
 	grp_priv = crt_grp_pub2priv(tgt_ep->ep_grp);
 
+	dst_tag = tgt_ep->ep_tag;
+
+	if (!crt_gdata.cg_provider_is_primary) {
+		/*
+		 * TODO: Add API to set number of destination tags
+ 		 * for the secondary provider
+ 		 */
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs